### PR TITLE
[doc] Improve operators page

### PR DESCRIPTION
**Pull request by lin-hitonami**
Related issue = #

### CODE REVIEW

**Review by qiao-bo — APPROVED** [review]
LGTM! Thanks

**Review by strongoier — APPROVED** [review]
LGTM!

File: docs/lang/articles/basic/operator.md [hunk 1]
```diff
@@ -67,10 +79,44 @@ default numerical types.
...
+@ti.kernel
+def cond_expr(ind: ti.i32) -> ti.i32:
+    return a[ind] if ind < 10 else 0
+
+cond_expr(3)  # returns 3
+cond_expr(10)  # returns 0, a[10] is not evaluated
+```
+
+
+For element-wise conditional operation on Taichi vectors and matrices,
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/basic/operator.md:112]:
```suggestion
For element-wise conditional operations on Taichi vectors and matrices,
```

File: docs/lang/articles/basic/operator.md [hunk 2]
```diff
@@ -136,9 +182,16 @@ ti.random(dtype=float)
 
 :::note
 
-The return number has a range that is type dependent. For example, floating
-point type returns a number between 0 and 1, while integer type such as `ti.i32`
-returns a number between -2147483648 and 2147483647.
+`ti.random` supports `u32`, `i32`, `u64`, `i64`, and all floating point types.
+The return value has a range that is type dependent.
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/basic/operator.md:186]: The range of the returned value is type-specific.

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]

**Comment by lin-hitonami** [comment]
Don't merge before the next version releases.

**Review by qiao-bo — APPROVED** [review]
One tiny nit: it's better to split the CPU/CUDA into two columns in the table, as we did in the type system page ;/ Other than that, looks great, thanks!

**Pull request merged**